bitsofgood/pencils-for-success
# Add script to seed data

### Description
We can use the script `npm run seed:dev` to seed data in the database. It uses prisma to upsert data into the database if it does not exist already.

**Related Issues/PRs:** 

- Closes #26 

### Test Plan

List of steps to test your proposed changes:
- Run `npx prisma db push --force-reset` to reset the existing database (if applicable).
- Run `npm run seed:db`.
- Open Prisma Studio to ensure the data for following models are created:
  - User
  - Admin User
  - Chapter
  - Chapter User
  - Recipient
  - Recipient User
- Run `npm run seed:db` to ensure duplicate records are not created.
